solidstart-tanstackquery-tailwind-modules/src/components/RepoReadMe/RepoReadMe.tsx

```diff
@@ -34,26 +40,29 @@ const RepoReadMe = () => {
   });
 
   return (
-    <>
-      {readme() ? (
-        <div class={styles.container} data-testid="readme">
-          <header class={styles.header}>
-            <span class={styles.tocIconContainer}>
-              <TocIcon class={styles.tocIcon} />
-            </span>
-            <span class={styles.filename}>README.md</span>
-          </header>
-          <article class={styles.article}>
-            <SolidMarkdown
-              rehypePlugins={[rehypeRaw, remarkGfm]}
-              children={readme()}
-            />
-          </article>
-        </div>
-      ) : (
-        <>{!pathname.includes('tree') ? <Empty /> : null}</>
-      )}
-    </>
+    <Show when={!resReadMe.isLoading && resReadMe.data}>
```

Comment:
In order to be more consistent maybe here it is more "correct" to check for `resReadMe.isSuccess && !resReadMe.isLoading`. We can check `resReadMe.data` in the `createEffect` before setting the signal...